org.eevolution.warehouse/src/main/java/base/org/spin/form/OutBoundOrder.java

```diff
@@ -236,10 +260,12 @@
 			// role security
 		} else {//Query for Sales Order
 			sql = new StringBuffer("SELECT " +
-					"wr.Name Warehouse, ord.C_Order_ID, ord.DocumentNo, " +	//	1..3
-					"ord.DateOrdered, ord.DatePromised, reg.Name, cit.Name, sr.Name SalesRep, " +	//	4..8
-					"cp.Name Partner, bploc.Name, " +	//	9..10
-					"loc.Address1, loc.Address2, loc.Address3, loc.Address4, ord.C_BPartner_Location_ID, ord.Weight, ord.Volume " +	//	11..17
+					"ord.C_Order_ID, ord.DocumentNo, ord.Description , cp.Name Partner, bploc.Name, wr.Name Warehouse , " +
```

Comment:
I don't sure if this is correct, maybe is a change for a customer or own change, the Weigh and Volume is showed from Order to select instead of Quantoty to deliver.

I think that this change can be part of a ADempiere Labs, Customizxation for your customer otr a Location